apps/view/src/images/types.d.ts

```diff
@@ -0,0 +1,9 @@
+declare module '*.png' {
+  const href: string
+  export = href
```

Comment:
I think I should prefer `export default` for ES6 modules (which is what webpack will be treating these as)